tests/async_engine/test_chat_template.py

```diff
@@ -62,12 +62,8 @@ class MockServingChat:
 
 def test_load_chat_template():
```

Comment:
I think we can delete `MockTokenizer` and `MockServingChat` since they aren't used anymore.